CONTRIBUTING.md

```diff
@@ -18,13 +18,36 @@ Welcome! We're so excited that you're considering contributing to the official G
 ---
 
 ## Commit your update
+Please follow our project commit guide lines below:
+- The first line is the title of the message,
+it should be no longer than 63 characters.
+- The second line should be left empty.
+- The remaining consist of the body of the message,
+they should be no longer than 72 characters each.
+try to be descriptive as you can while keeping it simple.
+
+- Example:
+Adding an awesome feature.
+
+Adding feature which:
+1. makes cookies.
+2. makes you get A+ in all courses.
+3. create dank memes.
+in order to use each of the feature above just go to page ****,
```

Comment:
No need for example here i think since all details has been written

Reply:
Please notice that you commented on lines 30-36 (I think you meant to mark lines 29-30).